grype/presenter/cyclonedx/vulnerability.go

```diff
@@ -36,17 +36,17 @@ type Source struct {
 
 // Rating has information about the intensity of a vulnerability
 type Rating struct {
-	Score    Score  `xml:"v:score"`
+	Score    Score  `xml:"v:score,omitempty"`
 	Severity string `xml:"v:severity,omitempty"`
 	Method   string `xml:"v:method,omitempty"`
 	Vector   string `xml:"v:vector,omitempty"`
 }
 
 // Score provides the different ways to measure how serious a vulnerability is
 type Score struct {
-	Base           float64 `xml:"v:base"`
-	Impact         float64 `xml:"v:impact"`
-	Exploitability float64 `xml:"v:exploitability"`
+	Base           float64 `xml:"v:base,omitempty"`
```

Comment:
I'm a little torn on making `Base` optional, though the [cycloneDX vuln extension spec](https://cyclonedx.org/schema/ext/vulnerability/1.0) does list it as optional, if a score (the struct) is provided at all, the base score is semantically non-optional.